src/cmake-build-debug/lib_ccx/cmake_install.cmake

```diff
@@ -0,0 +1,96 @@
+# Install script for directory: /Users/Gurpreet/Desktop/CCextractor/src/lib_ccx
```

Comment:
We don't need any file from the cmake-build-debug folder, that's not any source code.